_posts/2021-06-17-sandstorm-community-fundraising.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Fundraising for the Sandstorm Community"
+author: Ian Denhardt
+authorUrl: https://github.com/zenhack
+---
+
+Hey Everyone!
+
+Members of the Sandstorm community are doing some fundraising!
+
+Specifically,
+
+- In the immediate term, we're [participating in FundOSS][1], a donation
+  matching program that is a joint effort between [OpenCollective][2]
```

Comment:
I'm a little fuzzy on exactly which orgs are involved in putting this together in the first place, @ocdtrekkie can you comment on whether I should add to/amend this?

Reply:
Open Source Collective is the org partnering with GitCoin. I think it's operated by the OpenCollective folks, but technically OC is the platform and OSC is the fiscal host which is participating in creating FundOSS.